MapifyEditor/Export/Validators/Project/MapInfoValidator.cs

```diff
@@ -28,6 +29,14 @@ protected override IEnumerator<Result> Validate(Scenes scenes)
             if (mapInfo.name == Names.DEFAULT_MAP_NAME)
                 yield return Result.Error($"Your map name cannot be {Names.DEFAULT_MAP_NAME}");
 
+            //Loading Screen
+            //LoadingScreenImages will be null if the map was built with an older version of Mapify
```

Comment:
This comment doesn't make sense since it's in the exporter. Was it supposed to say something along the lines of `LoadingScreenImages will be null if none are configured`? I don't recall if Unity would make the array null or empty if it's unset.

Reply:
> This comment doesn't make sense since it's in the exporter

Not if the map was exported with an older version of Mapify.

Reply:
Oops, you're right, I got confused and commented this in the wrong place.